replica/replica.go

```diff
@@ -1155,6 +1160,12 @@ func (r *Replica) GetCloneStatus() string {
 	return info.CloneStatus
 }
 
+func (r *Replica) GetPreloadStatus() types.PreloadStatus {
```

Comment:
Do we need this?

Reply:
yes, adding this in replica info. Will be good for observability purpose.